src/Deanery.java

```diff
@@ -0,0 +1,111 @@
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Scanner;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+
+public class Deanery {
+    static DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss");
+    static LocalDateTime now = LocalDateTime.now();
+
+    private static String createRaport(Student student, String roomNumber) {
+        return "------------------------------------------" +
+                "\n" +
+                String.format("%20s", "STUDENT ") + student.getID().toString() +
+                "\n" + "\n" +
+                "First Name" + String.format("%10s", " ") + "|" + String.format("%3s", " ") + student.getName() + "\n" +
+                "Second Name" + String.format("%9s", " ") + "|" + String.format("%3s", " ") + student.getSecondName() + "\n" + "\n" +
+                "Raport generated on " + dtf.format(now) + "\n" +
+                "Room number: " + roomNumber +
+                "\n" +
+                "------------------------------------------";
+    }
+
+    public static void serve(Student student, String roomNumber, int hour) {
+        if (!(roomNumber.equals(Rooms.A123.toString())||roomNumber.equals(Rooms.B123.toString())||roomNumber.equals(Rooms.C124.toString())||roomNumber.equals(Rooms.D124.toString()))) {
```

Comment:
you could use te Rooms.values() method to retrieve all values of the Rooms enum and use some loop iteration here for example -> because if we had 100 rooms that would get a biiiiit of a long line